docs/GettingStartedDocs/VisualStudioLinux.md

```diff
@@ -0,0 +1,147 @@
+   - On the Configure Networking screen, choose Default Switch to ensure you can connect to it with a debugger.
+   - On the Installation Options screen, choose the ISO file you downloaded.
+   - All other options can be either left as the defaults or changed as desired.
+1. Disable Secure Boot as follows.  In Hyper-V Manager, right click on the VM you created while it is stopped,
+  and select Settings... -> Security and uncheck Enable Secure Boot.
+1. Uncheck "Enable checkpoints" under the VM's Settings -> Checkpoints, since SGX will not work with checkpoints.
+1. If using an SGX-capable machine, enable SGX for the VM as follows (this cannot be done from Hyper-V Manager):
+   - Download [VirtualMachineSgxSettings.psm1](https://raw.githubusercontent.com/microsoft/openenclave/f28cedce63be9673e20fe54563987189f2565637/new_platforms/scripts/VirtualMachineSgxSettings.psm1)
+   - Open an elevated PowerShell window (e.g., type "powershell" and click Run as Administrator)
+   - Invoke the following commands, using the path to where you downloaded the file, and replacing MyVM with your VM name:
+   ```
+   Set-ExecutionPolicy Bypass -Scope Process
+   Import-Module Drive:\Path\to\VirtualMachineSgxSettings.psm1
+   Set-VMSgx -VmName MyVM -IsSgxEnabled $True -SgxSize 32
+   ```
+1. Start the VM and connect to it (right click, Connect...), finish the initial setup, reboot, and login.
+   - Enable OpenSSH server installation when given the choice during setup.
+   - All other options are sufficient to leave as the defaults or changed as desired.
+
+On the Linux build machine, or after opening an ssh session into the VM:
+
+- Install the Open Enclave SDK.  See [installation instructions for Ubuntu 16.04](https://github.com/microsoft/openenclave/blob/master/docs/GettingStartedDocs/install_oe_sdk-Ubuntu_16.04.md)
+  or [installation instructions for Ubuntu 18.04](https://github.com/microsoft/openenclave/blob/master/docs/GettingStartedDocs/install_oe_sdk-Ubuntu_18.04.md), except that step 2 on those pages is outdated and result
+  in SGX not working.  Instead, replace step 2 with the instructions
+  [here](https://github.com/microsoft/openenclave/blob/master/docs/GettingStartedDocs/Contributors/SGX1GettingStarted.md)
+  prior to the Install section, which should work with either 16.04 or 18.04 even though
+  the page only mentions 16.04.
+
+Finally, configure Visual Studio with the address (or name) of your Linux build machine,
+via Tools -> Options -> Cross Platform -> Connection Manager -> Add. This step may take
+a minute or two, as Visual Studio will copy some files locally for use by IntelliSense.
+
+## Walkthrough: Creating a C/C++ Enclave Application
+
+We will now walk through the process of creating a C/C++ application that uses an enclave.
+
+1. Create a new Linux application using File -> New -> Project and find the Linux console
+   app template.  In VS2017, this is called "Console Application (Linux)", and in VS2019,
```

Comment:
In my install, "Console Application (Linux)" exists under "Installed->Other Languages -> Visual C++ -> Cross Platform -> Linux". This was slightly hard to find since i had installed only the Linux workload and didn't expect C++ to be an "other language" or expect Linux stuff to be under "Visual C++".

Consider whether specifying the full path would be more user friendly.